src/app/feature-flag/notification-footer/feature-footer.component.less

```diff
@@ -0,0 +1,73 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
+.feature-footer {
```

Comment:
this produces Stylelint errors - you need to have spaces after each attribute. i.e. `position: absolute;`

run `npm run stylelint` to see the errors